microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -206,6 +206,38 @@ chrome.tabs.onActivated.addListener(async ({ tabId }) => {
 ```
 
 
+<!-- ------------------------------ -->
+#### Opening the sidebar upon user interaction
+<!-- todo: note: new section.
+other doc: https://developer.chrome.com/docs/extensions/reference/sidePanel/#user-interaction -->
+
+[sidePanel.open()](https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-open) allows extensions to open the sidebar through an user gesture, such as clicking on the action icon.  Or any user interaction on an extension page or content script, such as clicking a button. 
+
+The following code shows how to open a global sidebar on the current window when the user clicks on a context menu.  When using `sidePanel.open()`, choose the context in which it should open.  Use `windowId` to open a global sidebar. 
+
+`sidePanel.open()` is available starting with Microsoft Edge version 117.
+
+Alternatively, set the `tabId` to open the sidebar only on a specific tab.
+
+```js
+// service-worker.js:
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.contextMenus.create({
+    id: 'openSidePanel',
```

Comment:
```suggestion
    id: 'openSidebar',
```
tech review this change of variable name; it would have to be renamed consistently.  2 hits in this .md file, both are in this listing.

Reply:
keeping as-is

Reply:
Yes, changing these two hits (on lines 224 and 231) should work, if we still want to do it.